Formula/nrpe.rb

```diff
@@ -40,7 +40,7 @@ def install
 
     inreplace "src/Makefile" do |s|
       s.gsub! "$(LIBEXECDIR)", "$(SBINDIR)"
-      s.gsub! "$(DESTDIR)/usr/local/sbin", "$(SBINDIR)"
+      s.gsub! "$(DESTDIR)#{HOMEBREW_PREFIX}/sbin", "$(SBINDIR)"
```

Comment:
This is an `inreplace`. The first argument of `s.gsub!` has to accurately reflect the contents of `src/Makefile` because this is the content of `src/Makefile` that we want to change. (i.e. we want to change anything that says `$(DESTDIR)/usr/local/sbin` in `src/Makefile` into `$(SBINDIR)`)

This works when Homebrew is installed on an Intel machine in the default prefix, but will break source builds everywhere else.